### DISCUSSION TITLE
Jonah 🐷. Version 1:

### DISCUSSION THREAD

**Discussion by MilesWK** [MilesWK/Jonah#4] [Announcements]
# Jonah 🐷. Version 1:
Features:
- Stable hosting
- Information 📕, Nuke 💣, and Me 😄 slash commands
- Talking 💬 
- A nice, piggy logo 🐽 
- A website 🖥️ 

I need ideas! Post any you have in the Ideas section of the Discussions!

🐷 Oink!

<hr /><em>This discussion was created from the release <a href='https://github.com/MilesWK/Jonah/releases/tag/Version'>Jonah 🐷. Version 1:</a>.</em>

**Comment by The-Best-Codes** [comment]
Is there a dedicated server with Jonah in it? :)

**Comment by MilesWK** [comment]
Kind of.... not really. I just created a "public" server so I could check
that box. Everything is off limits lol.

I also have like 0 users so...

**Reply by The-Best-Codes** [reply]
Yeah, I saw some of your posts on dev.to, so I was checking it out. :grinning:

**Reply by MilesWK** [reply]
Thanks! I am glad someone actually checked it out!
